examples/organization-viewer-cli/package.json

```diff
@@ -7,7 +7,13 @@
     "build": "yarn clean && yarn compile",
     "dev": "yarn build --watch",
     "clean": "rm -rf ./dist ./tsconfig.tsbuildinfo",
-    "start": "node dist/src/index.js",
```

Comment:
I split the cli demo into multiple demos because it was becoming unmanageable. I like the new separation a lot because going into each specific demo makes it very clear. It should also allow us to further simplify them because their setup no longer needs to be so generic.

Reply:
Nice, good idea!